decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb

```diff
@@ -39,7 +39,7 @@ def query
         end
 
         def proposals
-          @proposals ||= query.result.page(params[:page]).per(15)
+          @proposals ||= query.result(distinct: true).page(params[:page]).per(15)
```

Comment:
I think this deserves a test to prevent regression. It would also document the otherwise hidden bug.

Reply:
I don't understand you

Reply:
I meant we should probably test the bug to prevent it doesn't happen again in the future.